components/scrollable-container/README.md

```diff
@@ -96,6 +96,16 @@ To automatically show, hide and resize the scrollbar set the `automatic` attribu
 
 `<gameface-scrollable-container class="scrollable-container-component fixed-width" automatic>`
 
+## Events
```

Comment:
I found that there is no mentioned how you can scroll programmarly via the `scrollPos` setter. Can we add it to the docs? :D

Reply:
as an example I done it in my sample like that:
```
const scrollableContainer = this.scrollableContainerComponent.querySelector('.guic-scrollable-container');
    const diff = element.offsetTop + element.offsetHeight * 3 - scrollableContainer.offsetHeight;
    scrollableContainer.scrollTop = diff >= 0 ? diff + element.offsetHeight : 0;
    this.scrollableContainerComponent.onScroll();
```

to scroll the container so the element is visible :D

Reply:
Added